angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/widget-card.html

```diff
@@ -116,9 +95,9 @@
         </launch-button>
       </div>
 
-      <!-- For simple content portlets, show only an icon -->
+      <!-- For simple content portlets, show only an icon (RARELY USED) -->
```

Comment:
Could be worth saying why it's rarely used, which I think is because most simple content portlets have an external alternativeMaximizedUrl.